### DISCUSSION TITLE
SDK 1.9.0 Schematron Rules

### DISCUSSION THREAD

**Discussion by luispozoWS** [OP-TED/eForms-SDK#861] [Q&A] [answered]
Hello,

In the last couple of weeks we have asked some questions regarding our application and possible missing rules 

- https://github.com/OP-TED/eForms-SDK/discussions/818
- https://github.com/OP-TED/eForms-SDK/discussions/826
- https://github.com/OP-TED/eForms-SDK/discussions/843
- https://github.com/OP-TED/eForms-SDK/discussions/831

Could you confirm that schematron rules do not cover every possible scenario to generate an XML that avoids an XSD error ?

Best regards

**Comment by YvesJo** [comment, marked as answer]
Hi,
Schematron rules are only relevant for XSD conform instances. Generating an XSD conform instance is mainly an editor related concern.
